modules/RTC/ScreenObtainer.js

```diff
@@ -138,11 +138,11 @@ const ScreenObtainer = {
                         let audioConstraints = false;
 
                         if (screenShareAudio) {
-                            audioConstraints = { };
+                            audioConstraints = {};
 
-                            if (this._getAudioConstraints()) {
-                                audioConstraints.optional = {
-                                    ...this._getAudioConstraints()
+                            if (typeof this._getAudioConstraints() !== 'boolean') {
```

Comment:
Can you please avoid calling the function twice? Take the value and then work with it please.